blocks/results-list-block/package.json

```diff
@@ -36,5 +36,7 @@
     "@wpmedia/shared-styles": "*",
     "styled-components": "^4.4.0"
   },
-  "gitHead": "83fb8d6685958d85ddbfcfbf01d9a5c864cd6c95"
+  "gitHead": "83fb8d6685958d85ddbfcfbf01d9a5c864cd6c95",
+  "devDependencies": {},
+  "dependencies": {}
```

Comment:
dev and dependencies can be removed

Reply:
np.  they were added when I added the testing to this module, but I had to move it up to the root package.json.